tests/test_pipeline_mixin.py

```diff
@@ -314,7 +315,6 @@ def data(n):
         run_batch_test(pipeline, examples)
 
     @is_pipeline_test
-    @require_torch
```

Comment:
to allow to run `test_modeling_tf_wav2vec2.py -k "test_pipeline_audio_classification"`